lib/routemaster/resources/rest_resource.rb

```diff
@@ -37,8 +37,9 @@ def update(id=nil, params)
       end
 
       def destroy(id=nil)
-        # no response wrapping as DELETE is supposed to 204.
-        @client.delete(@url.gsub('{id}', id.to_s))
+        @client.with_response(Responses::HateoasResponse) do
```

Comment:
I think we never want to wrap the response to `DELETE` because our API guidelines say there should never be a body :)

(ie. the comment was valid)